src/pages/People/PeoplePage.tsx

```diff
@@ -0,0 +1,131 @@
+import { useContext, useEffect } from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Loader } from '../../components/Loader';
+import { PeopleList } from '../../components/PeopleList/PeopleList';
+import { PeopleContext } from '../../peopleContext';
+import { PeopleFilters } from '../../components/PeopleFilters/PeopleFilters';
+import { getSearchWith } from '../../utils/searchHelper';
+import { SortingFields } from '../../types/SortingFields';
+import classNames from 'classnames';
+
+export const PeoplePage = () => {
+  const [searchParams] = useSearchParams();
+  const {
+    loader,
+    warning,
+    setLoader,
+    setPeople,
+    setWarning,
+    setCenturiesArray,
+  } = useContext(PeopleContext);
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  useEffect(() => {
+    if (warning) {
+      setWarning('');
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [searchParams]);
```

Comment:
looks strange
Consider refactor the logic

Reply:
I don't no, how can i do it another way.